src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_AzDoFlow.cs

```diff
@@ -0,0 +1,23 @@
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    [Category("PostDeployment")]
+    class ScenarioTests_AzDoFlow : MaestroScenarioTestBase
+    {
+        [Test]
+        public async Task Darc_AzDoFlow_Batched()
+        {
+            TestParameters parameters = await TestParameters.GetAsync();
+            SetTestParameters(parameters);
+
+            EndToEndFlowLogic testLogic = new EndToEndFlowLogic(parameters);
+
+            TestContext.WriteLine("Azure DevOps Dependency Flow, batched");
+            await testLogic.DarcBatchedFlowTestBase($"AzDo_BatchedTestBranch_{Environment.MachineName}", $"AzDo Batched Channel {Environment.MachineName}", true);
```

Comment:
It's... odd that all of this tests is actually just hiding in some other method.  There is almost nothing interesting in this test.

Why isn't that method itself the test?

Reply:
Because it shares code with the GitHub Batched Flow test.

Reply:
There is almost no unique code to this test. It feels like the code in "DarcBatchedFlowTestBase" should just BE the test.  This method doesn't perform any uniquely interesting setup, and has no assertions in it at all.

Reply:
It's basically set up as a cheap data-driven test so that it can be used for both GitHub & AzDo. I agree that it makes a little less sense than the Non-Batched tests (where there are a lot of inputs).